contrib/openstack/config.sh

```diff
@@ -0,0 +1,55 @@
+export DEIS_TEST_DOMAIN="xip.io"
+export GLANCE_IMAGE
+export FLAVOR
+export KEY
+export NETWORK
+export POOL
+export OS_USERNAME
+export OS_PASSWORD
+export OS_TENANT_NAME
+export OS_AUTH_URL
+
+prompt "What glance image should I use? [coreos-stable]" GLANCE_IMAGE
+prompt "What flavor ? [2]" FLAVOR
+prompt "Path to your public key: [~/.ssh/id_rsa.pub] " KEY
+prompt "What is the name of your neutron network? [internal]" NETWORK
+prompt "What is the name of your floating IP pool ? [external]" POOL
+prompt "what is your openstack auth URL? [${OS_AUTH_URL}]" OS_AUTH_URL
+prompt "what is your openstack username? [${OS_USERNAME}]" OS_USERNAME
+prompt "what is your openstack password? [**********]" OS_PASSWORD
+prompt "what is your openstack tenant name? [${OS_TENANT_NAME}]" OS_TENANT_NAME
+
+export TF_VAR_deis_root="${DEIS_ROOT}"
+export TF_VAR_image="${GLANCE_IMAGE}"
+export TF_VAR_flavor="${FLAVOR}"
+export TF_VAR_public_key_path="${KEY}"
+export TF_VAR_network_name="${NETWORK}"
+export TF_VAR_floatingip_pool="${POOL}"
+export TF_VAR_auth_url="${OS_AUTH_URL}"
+export TF_VAR_username="${OS_USERNAME}"
+export TF_VAR_password="${OS_PASSWORD}"
+export TF_VAR_tenant_name="${OS_TENANT_NAME}"
+export TF_VAR_stack_name="deis-${DEIS_ID}"
```

Comment:
You should be able to set most of these directly via the prompt function above. Then you just have to export them here.